### ISSUE TITLE
Library webforms

### ISSUE THREAD

**Issue by jasloe** [aarome/aarome#149]
@cchristopherhowardd:

This is the basic idea for library webforms. If possible, it'd be good to run it by library staff first, as we'll use this as a template for the three other forms. 

(I just saw the botched submit/reset buttons at the bottom....will fix that!)

**Comment by cchristopherhowardd** [comment]
Is there a link to the form?

**Comment by jasloe** [comment]
All links point to the Missing Book form at the moment. I just revised the style sheet in order to place title elements in the left margin. Will fix shortly. 

http://dev.aarome.org/research/library/request-forms

**Comment by jasloe** [comment]
Clarification: http://dev.aarome.org/research/missing-book 

is available. 

Once this is approved, I'll clone and modify this form.

Fixed janky styling.

**Comment by cchristopherhowardd** [comment]
Three of the four forms are producing a page like this.
<img width="901" alt="Screen Shot 2019-03-12 at 9 59 18 AM" src="https://user-images.githubusercontent.com/38765846/54205905-a2086180-44ad-11e9-9066-37fd55bcf165.png">

**Comment by cchristopherhowardd** [comment]
The Report a Missing Book form shows me this.
<img width="1355" alt="Screen Shot 2019-03-12 at 9 59 30 AM" src="https://user-images.githubusercontent.com/38765846/54205940-b3516e00-44ad-11e9-882f-69a2664324cc.png">

**Comment by jasloe** [comment]
Yeah that's not correct! 

I accidentally overwrote the stylesheet for that form! I'm not in my office today and will recover the stylesheet from my office workstation tomorrow. 

Regarding the other 3 forms, we wanted to take feedback on the Missing Book form from the library (or do we?) before moving forward with the other three forms.

**Comment by jasloe** [comment]
Actually...scratch that! I was able to recover the code from my git log! Whew.

https://dev.aarome.org/research/missing-book
https://dev.aarome.org/about/tours

**Comment by jasloe** [comment]
@cchristopherhowardd any feedback from the Library on the example form?

**Comment by cchristopherhowardd** [comment]
No, but the current forms don't quite align with what I sketched out in the attached document. For one, I think we're going with a single field for name instead of first name and last name. The Quick News submission form reflects this: https://dev.aarome.org/news/quick-news/submit. All library forms can ask for (1) name (2) email (3) affiliation across the board.

[Academy Web Forms.docx](https://github.com/aarome/aarome/files/3045104/Academy.Web.Forms.docx)

**Comment by cchristopherhowardd** [comment]
Do you recall if we're having a single form for English and Italian, or a page for each language?

**Comment by jasloe** [comment]
Segmented fields is better for the following reason: library staff searching/sorting inquiries by last name. 

I added Title/Author/Publisher, etc. fields to all library fields in order harmonize the forms. Also, figure more information for library inquiries is generally better. If you'd like to revise these let me know. 

Yes, we can easily have en/it forms. 

https://dev.aarome.org/it/ricerca/desiderata

Just a matter of translating the form elements. I'm pretty sure I have all of the necessary translations.
